pace-util/pace/util/report.py

```diff
@@ -0,0 +1,114 @@
+import copy
+import json
+from datetime import datetime
+from typing import Any, Dict, List
+
+import numpy as np
+
+from pace.util.mpi import MPI
+
+
+def set_experiment_info(
```

Comment:
The methods you added here don't have unit tests - can you add some?

Is there a reason to put these in `pace-util` or should they be put in `driver`? Some of these seem pretty specialized to timing the full model.

Reply:
Moved report.py to driver

Reply:
Added unit test: test_timing_info